diff --git a/dist/compiler-core.cjs.js b/dist/compiler-core.cjs.js
index cf62cb1935198efacd6c5e3b6260f960b7ef53ba..8158127c31a4283ce0b5e5222fed156496a6f262 100644
--- a/dist/compiler-core.cjs.js
+++ b/dist/compiler-core.cjs.js
@@ -1063,7 +1063,7 @@ class Tokenizer {
     this.buffer = input;
     while (this.index < this.buffer.length) {
       const c = this.buffer.charCodeAt(this.index);
-      if (c === 10) {
+      if (c === 10 && this.state !== 33) {
         this.newlines.push(this.index);
       }
       switch (this.state) {
diff --git a/dist/compiler-core.esm-bundler.js b/dist/compiler-core.esm-bundler.js
index 8da200ea7fe1b73a3735b578b42d326e99933cfe..06dfdef807808c2246f25acf99439f0d8a8db4a9 100644
--- a/dist/compiler-core.esm-bundler.js
+++ b/dist/compiler-core.esm-bundler.js
@@ -1023,7 +1023,7 @@ class Tokenizer {
     this.buffer = input;
     while (this.index < this.buffer.length) {
       const c = this.buffer.charCodeAt(this.index);
-      if (c === 10) {
+      if (c === 10 && this.state !== 33) {
         this.newlines.push(this.index);
       }
       switch (this.state) {
